i18n/pt-PT.js

```diff
@@ -58,16 +58,16 @@ export default {
       ui: 'UI',
       data: 'Dados',
       modules: 'Módulos',
-      deployment: 'Desdobramento'
+      deployment: 'Produção'
     },
-    cli: 'Com a CLI',
-    scratch: 'De raiz'
+    cli: 'Interface da Linha de Comando',
+    scratch: 'Desde o Zero'
   },
   modules: {
-    search: 'Procurar um módulo (nome, categoria, nome de utilizador, etc.)',
+    search: 'Procurar um módulo por (nome, categoria, nome de utilizador, etc.)',
```

Comment:
delete `por`?

Reply:
Is it really necessary?
I want to be more clear as possible,
in this case, I think it would be better if I use `pelo` instead of `por` or `através do`.

What do you think?